src/app.tsx

```diff
@@ -10,7 +10,7 @@ import { getPokemon } from "./services/pokemon";
 import "./app.css";
 
 export const App = () => {
-  const [items, setItems] = useState([]);
+  const [items, setItems] = useState<Pokemon[]>([]);
```

Comment:
This basically says that `items` is a list of `Pokemon` and if anything else gets set to it, it should error. For example it can't hold a random object or list of numbers.